workload/cli: add --init-conns for init INSERT

darinpp opened the pull request:
Previously the number of connections when doing workload init INSERT was
hardcoded to 16.
This isn't always optimal and in case of free-tier clusters, it leads to
timeouts on the server side (because of 10s default login timeout).
This PR now allows the caller to override the default 16 by specifying a
--init-conns XX flag.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/59506)
<!-- Reviewable:end -->

darinpp committed:
workload/cli: add --init-conns for init INSERT

Previously the number of connections when doing workload init INSERT was
hardcoded to 16.
This isn't always optimal and in case of free-tier clusters, it leads to
timeouts on the server side (because of 10s default login timeout).
This PR now allows the caller to override the default 16 by specifying a
--init-conns XX flag.

Release note: None

nvb reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/59506#-:-MS9z4fS9vfBmTF1PkwD:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->

darinpp commented:
bors r+
